docs/mkdocs/mkdocs.yml

```diff
@@ -212,7 +212,7 @@ nav:
         - '(Constructor)': api/json_pointer/json_pointer.md
         - 'back': api/json_pointer/back.md
         - 'empty': api/json_pointer/empty.md
-        - 'operator std::string': api/json_pointer/operator_string.md
+        - 'operator string_t': api/json_pointer/operator_string.md
```

Comment:
Shouldn't the name of the file be `operator_string_t.md`?

Reply:
Yes. I was considering adding `mkdocs-redirects` before making more changes to names and structure. What are your thoughts about making sure links on the web keep working?

Reply:
Yes, good idea.